packages/material-ui/src/AvatarGroup/AvatarGroup.js

```diff
@@ -2,36 +2,72 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import { isFragment } from 'react-is';
 import clsx from 'clsx';
-import { chainPropTypes } from '@material-ui/utils';
-import { withStyles } from '../styles';
+import { chainPropTypes, deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import Avatar from '../Avatar';
+import { getAvatarGroupUtilityClass } from './avatarGroupClasses';
 
 const SPACINGS = {
   small: -16,
   medium: null,
 };
 
-export const styles = (theme) => ({
+const overridesResolver = (props, styles) => {
+  return deepmerge(styles.root || {}, {});
```

Comment:
Missing style overrides for the avatar slot. You can use the avatarGroupClasses.avatar

Reply:
@mnajdova Is this the correct way of style override for avatar slot?

```suggestion
  return deepmerge(styles.root || {}, {
    ...styles.avatar,
  });
```

Reply:
```suggestion
  return deepmerge(styles.root || {}, {
     [`& .${avatarClasses.avatar}`]: styles.avatar,
  });
```

this is the correct way. We need to use the classes selector for that slot and apply the styles there